crates/serde/Cargo.toml

```diff
@@ -9,8 +9,8 @@ rust-version = { workspace = true }
 [dependencies]
 anyhow = { workspace = true }
 num-bigint = { workspace = true }
-pathfinder-common = { path = "../common" }
-pathfinder-crypto = { path = "../crypto" }
+pathfinder-common = { version = "0.15.3", path = "../common" }
```

Comment:
Why do these dependencies have a fixed version while others (i.e. in p2p) don't?

Reply:
Only crates that are published require explicit versioning for their dependencies